common/source/docs/common-aerotate-dronecan-battery.rst

```diff
@@ -0,0 +1,66 @@
+.. _common-aerotate-dronecan-battery:
+
+===========================
+Aerotate DroneCAN Battery
+===========================
```

Comment:
overline/underline must be same length as heading text

Reply:
Done